provider/clusterrosa/classic/cluster_rosa_classic_resource.go

```diff
@@ -1542,29 +1541,32 @@ func populateRosaClassicClusterState(ctx context.Context, object *cmv1.Cluster,
 			state.Proxy = &proxy.Proxy{}
 		}
 		httpProxy, ok := proxyObj.GetHTTPProxy()
-		if ok {
+		if ok && httpProxy != "" {
```

Comment:
Why using empty string for reset the value and not using null for this?
Before this change, what happen if user update the value from valid value to null?
`httpProxy = null`

Reply:
Nothing happened, plus the API usually omits null. It currently expects empty strings to reset